### PR TITLE
background-image added

### PR DESCRIPTION
**Pull request by nmemarcoding**
just new background image added to the html tag

### CODE REVIEW

**Review by hjohnsick — COMMENTED** [review]
I see the images added in the images folder, but I don't see the images actually being utilized in the code.  When I view the site, I don't see any background image.

File: assets/css/style.css [hunk 1]
```diff
@@ -91,4 +108,11 @@ h2,
 
 #error-header {
     background-color: #E82F1C;
-}
\ No newline at end of file
+}
+
+
+/* @media all and (max-width: 768px) {
```
Comment (hjohnsick) [inline assets/css/style.css]: Can pry remove this since its commented out.

**Review by hjohnsick — APPROVED** [review]
Looks awesome!

File: assets/css/style.css (hunk 1 above)
Comment (nmemarcoding) [reply assets/css/style.css]: fixed

**Pull request merged**